assets/in

```diff
@@ -63,7 +63,11 @@ git clone --single-branch $depthflag $uri $branchflag $destination
 cd $destination
 
 git fetch origin refs/notes/*:refs/notes/*
-git checkout -q $ref
+until `git checkout -q $ref`; do
+    deepen=$((deepen+$depth))
```

Comment:
This only makes sense to attempt if `depth` is given (it defaults to 0).

Reply:
:+1: